test/unit/org/apache/cassandra/io/sstable/LegacySSTableTest.java

```diff
@@ -596,6 +637,13 @@ private static void loadLegacyTable(String tablePattern, String legacyVersion)
             }
         }
 
+        // if reading mc or lower format sstables, behave like upgrading from a CassandraVersion 3.0
+        if (legacyVersion.compareTo("mc") <= 0)
+            FBUtilities.setPreviousReleaseVersionString("3.0.8");
+
+        // mimic what StorageService does
+        //FIXME – SSTableHeaderFix.fixNonFrozenUDTIfUpgradeFrom30();
```

Comment:
todo

production is going to run this when upgrading from a Cassandra 3.0 cluster, and it breaks.

either we figure out why CC is now breaking it, or forbid upgrading to HCD-1.1 from C* 3.0.x

Reply:
ref:  https://datastax.slack.com/archives/C025YAXGFPA/p1732795482087489